### PR TITLE
Translation: Minor modifications and update

### PR DESCRIPTION
**Pull request by buhtz**
Let me give you a tip for review: Ignore the content in the "po" folder and just have a short look at "messages.pot" their.

Some minor things I discovered while working on another topic.

- Add warning comment about `_('Take snapshot')` source string (see #1559)
- Improved source strings in _Auto-Remove_ tab of the settings dialog ([mailing list discusson](https://mail.python.org/archives/list/bit-dev@python.org/message/A3YWD4WSBTC4GX3IY5QN4Y2EQWFX6ZR6/)). See screenshot below.
- Improved sources strings in _Question Dialog about restoring files_.
- ~Merged sources strings `Error` and `Errors`. Reasons: This does prevent "Reused translation" errors in our translation system. Singular and Plural in German is similar "Fehler". --> **After investigating languages with multiple plural forms (e.g. Arabic, Polish) I'm sure this solution is not elegant enough.**~
- Fix a minor visual bug in language dialog.
- Sync between upstream po(t) files and Weblate platform.

Here is the new _Auto-Remove_ tab:
image

Here is the new _Restore Question_ in singular form:
image

### CODE REVIEW

**Comment by buhtz** [comment]
After doing some research in the translation community I'm not convinced that using `ngettext()` for "Error" and "Errors" is a good solution. The reason is that there are languages with multiple plural forms (e.g. Arabic and Polish). It seems to me that `ngettetx()` make sense only when used with a concrete number/count. But in our situation we do use "Errors" here in the meaning of "All errors", which is the plural-form "Others" in Arabic and "Few" in Polish.

I'm doing further research for a more elegant solution to prevent "reused translation" errors.

*EDIT*: It might be the easiest to add `ignore-reused` as a flag to the source string via the Weblate platform. I have to investigate if this flag persists after using our github-weblate-sync process.

File: qt/logviewdialog.py [hunk 1]
```diff
@@ -89,13 +89,18 @@ def __init__(self, parent, sid = None, systray = False):
...
         self.comboFilter.addItem(_('All'), 0)
+
+        # Note about ngettext plural forms: n=102 means "Other" in Arabic and
+        # "Few" in Polish.
+        # Research in translation community indicate this as the best fit to
+        # the meaning of "all".
         self.comboFilter.addItem(' + '.join((_('Errors'), _('Changes'))), 4)
         self.comboFilter.setCurrentIndex(self.comboFilter.count() - 1)
-        self.comboFilter.addItem(_('Errors'), 1)
+        self.comboFilter.addItem(_('Errors'))
```
Comment (aryoda) [inline qt/logviewdialog.py]: I didn't dig deeper but why remove the "1" as argument if the other `addItems` do need this argument?

**Review by aryoda — APPROVED** [review]

File: qt/logviewdialog.py (hunk 1 above)
Comment (buhtz) [reply qt/logviewdialog.py]: I have not noticed. You are right that is a mistake. The `1` is the value of the combo box entry and has nothing to do with GNU gettext.

**Comment by buhtz** [comment]
I double checked the diff again.

**Pull request merged**